tasks/section5.yml

```diff
@@ -95,7 +95,8 @@
       - patch
       - rule_5.1.8
 
-- stat:
+- name: stat at.allow
```

Comment:
I'm for adding an actual name to this task but lets use the name of the CIS check that it relates to? i.e.
`"SCORED | 5.1.8 | PATCH | Check if at.allow exists"`

Reply:
:+1: